### PR TITLE
Implement Requery option and requery when calling customQueryHotkey

### LINKED ISSUE #701: results not refreshing if keyword is the same as last time
I set `ctrl+shift+c` to trigger `cb` keyword, which shows clipboard history.
When i press the hotkey, the window shows, but the results are not updated, I have to press an extra space to see current clipboard history.

**To Reproduce**
1. install clipboard history plugin, and assign a hotkey to trigger it.
2. press the hotkey, see current history.
3. copy some text.
4. press the hotkey again, the text just copied not show until input changes.

**Screenshots**

**Your System**
Windows 10

**Flow Launcher Error Log**

### PR DESCRIPTION
**Pull request by taooceros**
resolves #701

### CODE REVIEW

**Comment by jjw24** [comment]
@taooceros have you tested this sovles the issue 701?

I remember the requery doesnt work from the context menu, do you know if this also addresses it?

**Comment by taooceros** [comment]
> @taooceros have you tested this sovles the issue 701?
> 
> I remember the requery doesnt work from the context menu, do you know if this also addresses it?

Hmm I don't think so

File: Flow.Launcher/ViewModel/MainViewModel.cs [hunk 1]
```diff
@@ -285,9 +285,16 @@ public string QueryText
...
         /// <param name="queryText"></param>
-        public void ChangeQueryText(string queryText)
+        public void ChangeQueryText(string queryText, bool reQuery = false)
         {
-            QueryText = queryText;
+            if (QueryText!=queryText) 
+            {
+                QueryText = queryText;
+            }
+            else if (reQuery)
```
Comment (jjw24) [inline Flow.Launcher/ViewModel/MainViewModel.cs:294]: I feel like this shouldnt be an 'else if', they should be two seperate statements

Comment (taooceros) [reply Flow.Launcher/ViewModel/MainViewModel.cs:294]: Well actually, when QueryText is not equal to queryText, the requery will be triggered. Only when they are the same, we need manually requery.

Comment (jjw24) [reply Flow.Launcher/ViewModel/MainViewModel.cs:294]: ohhh it's requeried in the setter method...

Comment (jjw24) [reply Flow.Launcher/ViewModel/MainViewModel.cs:294]: lets add a comment in to make it clear

Comment (jjw24) [reply Flow.Launcher/ViewModel/MainViewModel.cs:294]: i will commit

Comment (taooceros) [reply Flow.Launcher/ViewModel/MainViewModel.cs:294]: I remember it is the optimization from fody xd

**Comment by taooceros** [comment]
the vscode online makes these kinds of quick commit much easier lol

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
i keep forgetting to use vscode online lol

**Pull request merged**